src/components/CalendarDay.jsx

```diff
@@ -64,7 +76,7 @@ export default class CalendarDay extends React.Component {
         onMouseLeave={e => this.onDayMouseLeave(day, e)}
         onClick={e => this.onDayClick(day, e)}
       >
-        {day.format('D')}
+        {this.renderDay(day)}
```

Comment:
@ljharb the linter requires that we have default props at every level, so we would have to use `renderDay: undefined` instead of `null` in the wrapping components, and it seems like that pattern is discouraged. I think the instance method is easier to reason about given that requirement.